config/routes.rb

```diff
@@ -101,6 +101,7 @@
 
     namespace :v3 do
       resources :fixed_streams, only: %i[show]
+      resources :stream_daily_averages, only: %i[show]
```

Comment:
Sorry, haven't realised that before, but I think it should be `index` not `show` as we return collection of `stream_daily_averages`.